packages/augur-sdk/src/state/create-api.ts

```diff
@@ -0,0 +1,51 @@
+import { Augur } from "../Augur";
```

Comment:
Should the UI be using this to create augur abject?

Reply:
Good question. I took your changes and moved them into a filename other than index. That was all I did re: crate and buildAPI.

Reply:
Looking further ... I don't know how to get the UI to use this stuff. There's a significant issue here in that Augur doesn't set the correct connector. It uses the EmptyConnector so these methods can only be used by backend stuff. It isn't too clean but I think this should be fixed in another PR.

Reply:
Not quite. This creates a provider. The UI might already have one injected. There could be some drying up to do here.

Reply:
Having a clear way for the UI to create the api object would be helpful. UI switches between creating without account address, with account address or switching account address.

Reply:
We can probably sugar this up for you but it is out of scope here.